### ISSUE TITLE
It can't be used.

### ISSUE THREAD

**Issue by boookk** [ysk24ok/jekyll-linkpreview#29]
Hi,

I used the tag within the markdown.

However, the tag content is displayed as it is and does not support preview.

Aren't we supposed to use tags within the markdown?

I want a detailed way to use it.

Thank you.

**Comment by ysk24ok** [comment]
@boookk Could you give me more details?

- The version of this plugin
- The version of Jekyll
- Way to reproduce
- etc

**Comment by boookk** [comment]
Thank you for your quick response.

I'm sorry for the late response.

jekyll-linkpreview (0.4.1)
jekyll (4.2.1, 3.9.1)

I installed the plug-in and placed the linkpreview.css file in the /assets/css directory.

And I wrote the phrase below in the head tag in the ./_include/head.html file.
```html
<link rel="stylesheet" type="text/css" href="/assets/css/linkpreview.css" media="screen">
```

After that, I inserted the link in the post using the tag as below.
```html
{% linkpreview "link" %}
```

As a result, gitaction fails, and tags and links appear as strings in the post.

However, it works when you run the local server and check the post.

What's my problem?
